.github/mergify.yml

```
@@ -14,12 +14,10 @@ pull_request_rules:
   - and:
     - label=end-of-life
     # Active ROS 1 distros
-    - label!=melodic
     - label!=noetic
     # Active ROS 2 distros
-    - label!=foxy
-    - label!=galactic
     - label!=humble
+    - label!=iron
```

Should we add Jazzy while we'ere here?
```suggestion
    - label!=iron
    label!=jazzy
```